src/crypto/crypto_cipher.cc

```diff
@@ -235,8 +235,19 @@ void CipherBase::GetSSLCiphers(const FunctionCallbackInfo<Value>& args) {
 
 void CipherBase::GetCiphers(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
+  MarkPopErrorOnReturn mark_pop_error_on_return;
```

Comment:
Do any of the OpenSSL calls produce errors?

Reply:
Yes, when you attempt to `_fetch()` an unsupported algorithm.

Reply:
Thanks, that wasn't clear from the OpenSSL docs.